K8s metrics helm (#552)
logzio/logz-docs: _source/logzio_collections/_metrics-sources/kubernetes.md

##### Run the automated deployment script

```shell
bash <(curl -s https://raw.githubusercontent.com/logzio/logzio-helm/master/quickstart-metrics.sh)
```

###### Prompts and answers

| Prompt | Description |
|---|---|

> | Logz.io metrics shipping token <span class="required-param"></span> | Replace `<<SHIPPING-TOKEN>>` with a [token](https://app.logz.io/#/dashboard/settings/manage-accounts)
> for the Metrics account you want to ship to. [Detailed instructions]({{site.baseurl}}/user-guide/accounts/finding-your-metrics-account-token/) |

Reviewer: ```suggestion
| Logz.io metrics shipping token <span class="required-param"></span> | Replace `<<SHIPPING-TOKEN>>` with a [token](https://app.logz.io/#/dashboard/settings/manage-accounts) for the Metrics account you want to ship to. [Detailed instructions]({{site.baseurl}}/user-guide/accounts/finding-your-metrics-account-token/) |
```

##### Store your cluster details

Paste the kube-state-metrics namespace in your text editor.
You can find it by running this command.

```shell
kubectl get service --all-namespaces | grep -E 'kube-state-metrics|NAMESPACE'
```

> Then find kube-state-metrics port under `http-metrics` when running this command.

Reviewer: ```suggestion
Next, run a command to find the kube-state-metrics port under `http-metrics`.
```

> Paste the kube-state-metrics namespace in your text editor.
> You can find it by running this command.
> 
> ```shell
> kubectl get service --all-namespaces | grep -E 'kube-state-metrics|NAMESPACE'
> ```
> 
> Next, run a command to find the kube-state-metrics port under `http-metrics`.
> 
> ```shell
> kubectl get service -n <KUBE-STATE-METRICS-NAMESPACE> kube-state-metrics -o yaml
> ```
> 

Reviewer: ```suggestion
Look up the following cluster details and copy them into your text editor or another convenient place for quick reference. 

Run the following command to get your <kube-state-metrics-namespace>.

```shell
kubectl get service --all-namespaces | grep -E 'kube-state-metrics|NAMESPACE'
```

Next, run a command to get your kube-state-metrics port. After you run the command, it will appear in the output under `http-metrics`. 

```shell
kubectl get service -n <KUBE-STATE-METRICS-NAMESPACE> kube-state-metrics -o yaml
```

```
